### ISSUE TITLE
bin_mean not found

### ISSUE THREAD

**Issue by aschetti** [ericcfields/FMUT#3]
Hi @ericcfields ,
many thanks for your work. I am testing FMUT on some old datasets (see this [GitHub repo](https://github.com/aschetti/test_FMUT)): in the main script [`test_FMUT.m`](https://github.com/aschetti/test_FMUT/blob/main/scripts/test_FMUT.m) I can successfully run `FmaxGND` on a 2x2x2 design. However, I encounter a peculiar issue when attempting to run pairwise contrasts: I cannot average over bins because I cannot locate the function `bin_mean`. I have looked for it in both the *Mass Univariate ERP* and *FMUT* folders, to no avail. I don't think it's a path issue, because all other FMUT functions in the script work properly. 

I hope I explained my issue clearly, thanks for any help you can provide.

**Comment by ericcfields** [comment]
`bin_mean` is a MUT function. See:

https://github.com/dmgroppe/Mass_Univariate_ERP_Toolbox/blob/master/bin_mean.m

If it isn't in your MUT folder, you should re-download the toolbox.

**Comment by aschetti** [comment]
Many thanks for your quick reply. I downloaded MUT from [GitHub](https://github.com/dmgroppe/Mass_Univariate_ERP_Toolbox) and can now locate `bin_mean`.

You and/or David Groppe might want to know that I encountered this issue when downloading MUT from EEGLAB (from the GUI, *Manage EEGLAB Extensions* --> *mass_univ v03272017*). Perhaps EEGLAB downloads an older version of MUT?

Anyway, thanks again for your help. I will now close this issue.
